package.json

```diff
@@ -7,22 +7,20 @@
   ],
   "scripts": {
     "lerna": "lerna",
-    "start": "lerna run start --stream --parallel",
+    "dev": "turbo run dev --parallel --no-cache",
     "test": "lerna run test --",
```

Comment:
Why do we need to have flags in the command - `turbo run test --ci --coverage`? I think `turbo run test` should be sufficient. If you want to use flags for a command, declare it inside each package instead.

Reply:
It's due to the existing github actions. We set it up as ` yarn test --ci --coverage` so I didn't want to modify that.

Reply:
Feel free to update that. Probably, create a script called `'test:ci' : 'yarn test --ci --coverage'` in each package then update the one in the Github action.

Reply:
👌Got it